samples/notebooks/dotnet/8-chatGPT-with-DALL-E-2.ipynb

```diff
@@ -0,0 +1,194 @@
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[An image of a friendly waving hand.]\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "var chat = (OpenAIChatHistory)chatGPT.CreateNewChat(\n",
+    "            \"You're chatting with a user. Instead of replying directly to the user\"\n",
+    "            + \" provide a description of an image that expresses what you want to say.\"\n",
+    "            + \" The user won't see your message, they will see only the image.\"\n",
+    "            + \" Describe the image with details in one sentence.\");\n",
+    "\n",
+    "while (true)\n",
+    "{\n",
+    "    // User enters a message\n",
+    "    var userMessage = await InteractiveKernel.GetInputAsync(\"Your message\");\n",
```

Comment:
Describe somewhere the difference between a user message, assistant message, and system message.